cpp/src/arrow/compute/api_scalar.cc

```diff
@@ -228,6 +268,12 @@ static auto kRoundOptionsType = GetFunctionOptionsType<RoundOptions>(
 static auto kRoundToMultipleOptionsType = GetFunctionOptionsType<RoundToMultipleOptions>(
     DataMember("multiple", &RoundToMultipleOptions::multiple),
     DataMember("round_mode", &RoundToMultipleOptions::round_mode));
+static auto kRoundTemporalOptionsType = GetFunctionOptionsType<RoundTemporalOptions>(
```

Comment:
nit: another contributor just alphabetized these, so maybe try to preserve that (same goes below).

Reply:
Oh yeah, will do!

Reply:
Done.